package.json

```diff
@@ -34,7 +34,11 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "@mdx-js/react": "^2.3.0",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.16",
```

Comment:
@HinataKah0 here @shanpriyan is adding MUI Icons. Would MUI Icons also work for you?

Reply:
Yes, MUI Icons will also work for `EditLink`.

However, I tried finding the usages of `@fortawesome`, it looks like it's consistently used across article related components (including "View as JSON"). "View as JSON" uses `faRobot` which doesn't seem to be available in MUI Icons. So it looks like we are going to need Font Awesome anyway unless we are changing others.

In this case, I would vote for keeping `EditLink` to use Font Awesome.

Reply:
I also see that MuiIcons don't have branded icons, such as GitHub Icon, which we need. @shanpriyan It might be better if we removed MUI Icons, and just keep fortawesome, or an Open Source alternative, but not sure if there's one, so I guess we can just use fortawesome.

Reply:
No, I'm talking about FortAwesome. Please double check.

FortAwesome is the company/org behind FontAwesome ;)

Reply:
How about [react-icons](https://react-icons.github.io/react-icons/search?q=github)? it does have branded icons like Github and Twitter icons

Reply:
Why not? :)

Reply:
@shanpriyan lemme know once you have an update here :)